.github/workflows/lint.yml

```diff
@@ -0,0 +1,15 @@
+name: Lint
+on:
+  push:
+jobs:
+  lint:
+    name: golangci-lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v4
+        with:
+          go-version: '1.17'
```

Comment:
Any reason for picking version `1.17` when we are using `1.19` rn?

We could also bump this version to `1.20` anyways, since there shouldn't be any breaking changes

Reply:
That was just some copy paste mistakes...